helm/hades/templates/ingress.yaml

```diff
@@ -1,8 +1,12 @@
 {{- if .Values.ingress.enabled -}}
+{{- $baseDomain := .Values.ingress.baseDomain | default "student.k8s.aet.cit.tum.de" -}}
+{{- $host := .Values.ingress.host | default (printf "%s.%s" .Release.Namespace $baseDomain) -}}
```

Comment:
I'm not sure if this is a good idea. I think hades should always be deployed to a concrete domain name, shouldnt it?